src/Microsoft.OpenApi.Hidi/Program.cs

```diff
@@ -35,7 +36,7 @@ static async Task<int> Main(string[] args)
             var formatOption = new Option<OpenApiFormat?>("--format", "File format");
             formatOption.AddAlias("-f");
 
-            var logLevelOption = new Option<LogLevel>("--loglevel", () => LogLevel.Warning, "The log level to use when logging messages to the main output.");
+            var logLevelOption = new Option<LogLevel>("--loglevel", () => LogLevel.Information, "The log level to use when logging messages to the main output.");
```

Comment:
Changed the default so that logs at the Information level will be showed by default. Without this the stats output by the validate are not shown by default.